### PR TITLE
add webpack profile due to pom change

### PR DESCRIPTION
**Pull request by Tcharl**
Following modifications from https://github.com/jhipster/generator-jhipster/pull/11207#discussion_r375192650

### CODE REVIEW

File: jhipster-daily-builds.iml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pascalgrimaud) [inline jhipster-daily-builds.iml]: can you delete this file and add *.iml to gitignore plz ?

File: test-integration/jdl-scripts/23-package.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ source $(dirname $0)/00-init-env.sh
 #-------------------------------------------------------------------------------
 if [[ "$JHI_APP" == *"uaa"* ]]; then
     cd "$JHI_FOLDER_UAA"
-    ./mvnw -ntp verify -DskipTests -Pdev
+    ./mvnw -ntp verify -DskipTests -Pdev,webpack
```
Comment (pascalgrimaud) [inline test-integration/jdl-scripts/23-package.sh]: no need this, as UAA doesn't have the front

File: azure-pipelines-official-windows.yml [hunk 3]
```diff
@@ -28,7 +28,7 @@ jobs:
   pool:
     vmImage: 'windows-2019'
   variables:
-    JHI_PROFILE: dev
+    JHI_PROFILE: dev,webpack
```
Comment (pascalgrimaud) [inline azure-pipelines-official-windows.yml]: ok, it should be ignore for microservice I think

**Comment by pascalgrimaud** [comment]
@SudharakaP : the GitBash build should not be triggered for each pull request. We need to deactivate it -> in fact, all configuration should not be triggered by PR

**Comment by SudharakaP** [comment]
@pascalgrimaud : Thanks for pointing it out. It took me a while to figure out we have to disable the CI triggers through the pipeline UI. I've done that, it shouldn't trigger on pull requests anymore. 😄 

image

**Comment by pascalgrimaud** [comment]
thanks @SudharakaP !

**Comment by SudharakaP** [comment]
You are welcome :smile:

**Pull request merged**